File: Jogador.java
public class Jogador{
    private final int cor;
    private final String nome;
    //falta inserir o vetor de peças

    public Jogador(String _nome , int _cor){
        this.cor = _cor;
        this.nome = _nome;
    }
    
    public void imprimir(){
        System.out.println(this.nome);
    }
}

File: jogador.cpp
/*
Projeto: Jogo de Xadrez
3º Semestre de Ciências da Computação - UFSCar Sorocaba
Programação orientada a objetos
Profª Katti Faceli
Integrantes: Giovanni Alvarenga, Fábio Yamaya, João Victor Vilar
Implementação da classe "jogador"
Descrição:Implementa as funções da classe "jogador" contidas no arquivo .h
*/
#include "jogador.h"
#include <string>

using namespace std;

//Construtor da classe ( ainda não utilizado )
Jogador::Jogador(string _nome, int _cor) : cor(_cor) {
    nome = _nome;
}

Jogador::~Jogador() {
    free(pecasAtivas);
}

//Imprime na tela o nome dos jogadores
void Jogador::printog() {
    cout << nome << endl;
}

Peca** Jogador::getPecasAtivas() {
    return pecasAtivas;
}
